virttest/env_process.py

```diff
@@ -1200,16 +1204,20 @@ def postprocess(test, params, env):
             err += "\nPB cleanup: %s" % str(details).replace('\\n', '\n  ')
             logging.error(details)
 
-    if err:
-        raise virt_vm.VMError("Failures occurred while postprocess:%s" % err)
     if params.get("verify_host_dmesg", "yes") == "yes":
         dmesg_log_file = params.get("host_dmesg_logfile", "host_dmesg.log")
         level = params.get("host_dmesg_level", 3)
         ignore_result = params.get("host_dmesg_ignore", "no") == "yes"
         dmesg_log_file = utils_misc.get_path(test.debugdir, dmesg_log_file)
-        utils_misc.verify_dmesg(dmesg_log_file=dmesg_log_file,
-                                ignore_result=ignore_result,
-                                level_check=level)
+        try:
+            utils_misc.verify_dmesg(dmesg_log_file=dmesg_log_file,
+                                    ignore_result=ignore_result,
+                                    level_check=level)
+        except exceptions.TestFail as details:
+            err += "\nHost dmesg verification failed: %s" % details
+
+    if err:
+        raise virt_vm.VMError("Failures occurred while postprocess:%s" % err)
```

Comment:
LGTM, except the type of the above error. It's a little bit misleading.

Reply:
Well I kept the original exception but I see your point. How about `RuntimeError` instead?

Reply:
OK, it's acceptable.